snippets/distr-tracing-tempo-tempomonolithic-custom-resource.adoc

```diff
@@ -19,14 +19,23 @@ spec:
       size: <value>Gi # <2>
       s3: # <3>
         secret: <secret_name> # <4>
+    tls:
+      caName: <ca_certificate_configmap_name> # <5>
   jaegerui:
-    enabled: true # <5>
+    enabled: true # <6>
     route:
-      enabled: true # <6>
+      enabled: true # <7>
+  resources: # <8>
+    total:
+      limits:
+        memory: <value>Gi
+        cpu: <value>m
```

Comment:
Can I include the `resources` section here?

Reply:
Yes, we can specify the resources.